python/ray/data/tests/test_all_to_all.py

```diff
@@ -326,6 +326,39 @@ def _to_pandas(ds):
     ]
 
 
+@pytest.mark.parametrize("num_parts", [1, 30])
+@pytest.mark.parametrize("ds_format", ["arrow", "pandas"])
+def test_groupby_multiple_keys_tabular_count(
+    ray_start_regular_shared, ds_format, num_parts, use_push_based_shuffle
+):
+    # Test built-in count aggregation
+    seed = int(time.time())
+    print(f"Seeding RNG for test_groupby_arrow_count with: {seed}")
+    random.seed(seed)
+    xs = list(range(100))
+    random.shuffle(xs)
+
+    def _to_pandas(ds):
```

Comment:
Wouldn't it be simpler to use `map_batches` for both pyarrow and pandas like [here](https://github.com/ray-project/ray/blob/master/python/ray/data/tests/test_sort.py#L88) instead of the if ds_format condition?